apps/sms/js/drafts.js

```diff
@@ -12,13 +12,7 @@
    *
    * A collection of active Draft objects, indexed by thread id.
    */
-  var Drafts = exports.Drafts = {
-    /**
-     * List
-     *
-     * An Array-like object that contains Draft objects.
-     */
-    List: DraftList,
```

Comment:
maybe a little bit of history would be useful: at first we were asked to support several drafts for one thread; they would show up in the list of messages. Then we were asked to support only one draft per thread because it was easier/was closer to what other platforms were doing/easier UX as well. So that's where this comes from, and we never really simplified it, mostly being afraid that the previous requirement might come back and because, well, this works :)

But now I'm all for simplifying this.